benchmarks/user/benchmark.py

```diff
@@ -322,8 +321,8 @@ def plot(problem, **kwargs):
              if len(set(dict(j)[i] for j in gflopss)) > 1]
 
     # Filename
-    figname = "%s_dim%s_so%s_to%s_arch[%s]_bkend[%s]_at[%s]pdf" % (
-        problem, shape, space_order, time_order, arch, backend, autotune
+    figname = "%s_arch[%s]_shape%s_so%s_to%s_bkend[%s]_at[%s]" % (
```

Comment:
there was a reason ``arch`` was next to ``bkend`` -- they both are "platform information", while ``shape/dim`` (good changing dim->shape), ``so``, ``to`` are about the problem executed

Reply:
Good, then we should change the order in produced .json files. In any case I was trying to have json and fignames homogeneous.